src/docs/asciidoc/gettingStarted.adoc

```diff
@@ -73,7 +80,7 @@ or the <<ref-maintenance-scripts-dbm-changelog-sync,dbm-changelog-sync>> script
 grails dbm-changelog-sync
 ----
 
-NOTE: If you use the searchable plugin you need to configure it to run after the database migrations have run. Either in `Searchable.groovy` or in `Config.groovy` (if that's where you configure Searchable):
```

Comment:
I forgot about the new fangled option to just fix it myself on your branch. I did so